examples/fortran/SHLocalizedAdmitCorr/meson.build

```diff
@@ -0,0 +1,12 @@
+TestLocalizedAdmitCorr = executable(
+  'LocalizedAdmitCorr',
+  'LocalizedAdmitCorr.f95',
+  link_with: shtools)
+
+test(
+  'LocalizedAdmitCorr',
+  TestLocalizedAdmitCorr,
+  args: [
+    meson.current_source_dir() / '../../ExampleDataFiles',
+    meson.current_source_dir() / 'input.txt',
+  ])
```

Comment:
This test timed out on my laptop. I would suggest adding this argument to `test`:
```
timeout: 60
```